Xamarin.Forms.Core/IControlTemplated.cs

```diff
@@ -8,6 +8,10 @@ internal interface IControlTemplated
 
 		IList<Element> InternalChildren { get; }
 
+		Element TemplateRoot { get; set; }
```

Comment:
IControlTemplated is internal. The setter is needed by `TemplateUtilities`. I prefer to continue using the `TemplateUtilities` as a mixin, instead of switching to duplicating code in `TemplatedView` and `TemplatedPage`